### ISSUE TITLE
Fail to build core/meta/src/TEnumConstant.cxx in gcc 11. Need to include <limits> in header file core/foundation/inc/ROOT/libcpp_string_view.h

### ISSUE THREAD

**Issue by yuxianch** [root-project/root#8513]
In gcc 11, core/meta/src/TEnumConstant.cxx cannot be built.
Error message:
```
/tmp/root-6.22.06/core/foundation/inc/ROOT/libcpp_string_view.h:275:63: error: ‘numeric_limits’ is not a member of ‘std’
  275 |         size_type max_size() const _NOEXCEPT { return (_VSTD::numeric_limits<size_type>::max)(); }
      |                                                               ^~~~~~~~~~~~~~
```
This is because <limits> is not explicitly included by core/foundation/inc/ROOT/libcpp_string_view.h. It can be built if the header is included.
https://github.com/root-project/root/blob/71e0b990b889ff292be5bfa79d1f85e15440eaa5/core/foundation/inc/ROOT/RWrap_libcpp_string_view.h#L40-L45

**Comment by vepadulano** [comment]
That include has been added to `libcpp_string_view.h` a while ago (see https://github.com/root-project/root/pull/8287 and backports down to 6.18). I wasn't aware we had so many string view headers, I will understand this better and include the header everywhere is needed

**Comment by vepadulano** [comment]
Ah wait, I now see that your error refers to `libcpp_string_view.h` indeed. But you're using 6.22/06, you will get no errors if you use master. The backport in 6.22 will be available in next patch release 6.22/10

**Comment by yuxianch** [comment]
Oh yes, it's fixed now. Thank you! Close this issue.
